.github/workflows/build-engine-branch.yml

```diff
@@ -18,17 +18,17 @@ jobs:
     steps:
       - name: Extract custom engine branch name from PR body
         id: detectCustomEngineBranch
+        env:
+          PR_BODY: ${{ github.event.pull_request.body }}
         if: |
           github.event_name == 'pull_request' &&
-          contains(github.event.pull_request.body, '/engine-branch') &&
+          contains(env.PR_BODY, '/engine-branch') &&
```

Comment:
For more info, see https://securitylab.github.com/research/github-actions-untrusted-input/